site/src/api/queries/workspaces.ts

```diff
@@ -124,3 +125,121 @@ export const increaseDeadline = (workspace: Workspace) => {
     },
   };
 };
+
+export const changeVersion = (
+  workspace: Workspace,
+  queryClient: QueryClient,
+) => {
+  return {
+    mutationFn: ({
+      versionId,
+      buildParameters,
+    }: {
+      versionId: string;
+      buildParameters?: WorkspaceBuildParameter[];
+    }) => {
+      return API.changeWorkspaceVersion(workspace, versionId, buildParameters);
+    },
+    onSuccess: async (build: WorkspaceBuild) => {
+      await updateWorkspaceBuild(build, queryClient);
+    },
+  };
+};
+
+export const update = (workspace: Workspace, queryClient: QueryClient) => {
+  return {
+    mutationFn: (buildParameters?: WorkspaceBuildParameter[]) => {
+      return API.updateWorkspace(workspace, buildParameters);
+    },
+    onSuccess: async (build: WorkspaceBuild) => {
+      await updateWorkspaceBuild(build, queryClient);
+    },
+  };
+};
+
+export const deleteWorkspace = (
+  workspace: Workspace,
+  queryClient: QueryClient,
+) => {
+  return {
+    mutationFn: ({ logLevel }: { logLevel?: ProvisionerLogLevel }) => {
+      return API.deleteWorkspace(workspace.id, logLevel);
```

Comment:
I just made a [couple of changes](https://github.com/coder/coder/pull/10654) to `deleteWorkspace`. Let me know if you need any help updating. Should be trivial.

Reply:
Ty for letting me know.